be/src/runtime/thread_context.h

```diff
@@ -111,10 +123,15 @@ class ThreadContext {
     }
 
     const std::string& task_id() const { return _task_id; }
-    const std::thread::id& thread_id() const { return _thread_id; }
-    const std::string& thread_id_str() const { return _thread_id_str; }
+    const std::string& thread_id_str() const { return _thread_id; }
     const TUniqueId& fragment_instance_id() const { return _fragment_instance_id; }
 
+    std::string get_thread_id() {
+        std::stringstream ss;
```

Comment:
better not using stringstream. use fmt instead

Reply:
stringstream is here just to convert `std::thread::id` to `std::string`, which fmt doesn't seem to be able to do.